src/base/logging.hh

```diff
@@ -123,19 +126,41 @@ class Logger
      */
     [[noreturn]] void exit_helper() { exit(); ::abort(); }
 
+    void
+    registerExtraLog(std::function<std::string()> cb)
+    {
+        extraLog.emplace_back(std::move(cb));
+    }
+
   protected:
     bool enabled;
 
+    std::string
```

Comment:
Ditto add documentation

Reply:
Added!

Reply:
nit: I know it is like this in the other functions (and I was the one that wrote the poor documentation in log()), but could you be more verbose? When adding documentation one should try to focus on information that is not acquired from the variable/function. For example, by reading the function name I know that this function will return the formatted extra log. But what is an extra log? What is the format?

Reply:
Thanks for the suggestion! I've added more description of this function. Please check :)